converter/shieldhit/geo.py

```diff
@@ -249,5 +251,7 @@ def get_mat_string(self) -> str:
         """Generate mat.dat config."""
         # we increment idx because shieldhit indexes from 1 while python indexes lists from 0
         material_strings = [
-            self.material_template.format(idx=idx+1, mat=mat[1])for idx, mat in enumerate(self.materials)]
+            self.material_template.format(idx=idx+1, mat=mat[1])
+            for idx, mat in enumerate(self.materials)
+            if mat[1] not in PREDEVINED_MATERIALS]
```

Comment:
You shouldn't need to do that, 0 and 1000 aren't valid ICRU's so they should not be in `self.materials`. Better to blacklist them in `converter.shieldhit.parser.ShieldhitParser._parse_materials`.